eventing/samples/github-source/README.md

```diff
@@ -114,58 +131,6 @@ Then, apply that yaml using `kubectl`:
 kubectl --namespace default apply --filename eventing/samples/github-source/github-source.yaml
 ```
 
-### Subscription
-
-To verify the `GitHubSource` is fully working, create a simple Knative
-`Service` that dumps incoming messages to its log and create a
-`Subscription` from the `Channel` to that Knative `Service`.
-
-If the deployed `GitHubEventSource` is pointing at a `Channel` other
-than `githubchannel`, modify `subscription.yaml` by replacing
-`githubchannel` with that `Channel`'s name.
-
-```yaml
-apiVersion: eventing.knative.dev/v1alpha1
-kind: Subscription
-metadata:
-  name: github-source-sample
-  namespace: knative-demo
-spec:
-  channel:
-    apiVersion: eventing.knative.dev/v1alpha1
-    kind: Channel
-    name: githubchannel
-  subscriber:
-    ref:
-      apiVersion: serving.knative.dev/v1alpha1
-      kind: Service
-      name: github-message-dumper
-
----
-# This is a very simple Knative Service that writes the input request to its log.
-
-apiVersion: serving.knative.dev/v1alpha1
-kind: Service
-metadata:
-  name: github-message-dumper
-  namespace: knative-demo
-spec:
-  runLatest:
-    configuration:
-      revisionTemplate:
-        spec:
-          container:
-            image: github.com/knative/eventing-sources/cmd/message_dumper
-
-```
-   
-Then, deploy `subscription.yaml`, creating both the `Service` and
-`Subscription`.
-
-```shell
-ko apply --filename eventing/samples/github-source/subscription.yaml
-```
-
 ### Create Events
```

Comment:
Can you verify the webhook before you create events?

Reply:
Done